pom.xml

```diff
@@ -13,13 +13,14 @@
   ~ limitations under the License.
   -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>uk.gov.gchq.magma-core</groupId>
   <artifactId>magma-core</artifactId>
-  <version>1.0</version>
-  <packaging>jar</packaging>
+  <version>1.1-SNAPSHOT</version>
```

Comment:
Consider updating the pom version here, and possibly in the `pom` for `core` before the merge, and maybe tag the current version of main as `1.0` so that people who depend on `main` can rebuild the correct version. Hopefully the publishing tools will manage tags for you in future in the way that `maven release` does. No need to reply to this, just go ahead with what you feel works.